CHANGES/7368.misc

```diff
@@ -0,0 +1 @@
+Derive ANSIBLE_CONTENT_HOSTNAME from CONTENT_ORIGIN
```

Comment:
The default of a variable I think needs a more user-facing changelog. Maybe even a removal type.